content/reference-transaction-format.md

```diff
@@ -157,6 +178,16 @@ After a transaction has been submitted, if it gets accepted into a validated led
 }
 ```
 
+
+### Multi-Signing ###
+
+Multi-signing in Ripple is the act of [authorizing transactions](#authorizing-transactions) for the Ripple Consensus Ledger by using a combination of multiple secret keys. Multi-signing is due to be enabled by an [Amendment](concept-amendments.html) to the Ripple Consensus Protocol. You can use multi-signing in addition to, or instead of, a master key, a [regular key](#setregularkey), or both.
+
+The [SignerListSet transaction](#signerlistset) defines which addresses can authorize transactions from your address. You can include up to 8 addresses in a SignerList. You can achieve several different configurations using the weight and quorum values of the SignerList.
```

Comment:
I don't want to use the word "permission" because I think that implies more fine-grained control over sending different types of transactions than multi-signing currently provides.

I went with this instead:
> You can control how many signatures are needed, in which combinations, by using the quorum and weight values of the SignerList.